docker/start_broker.sh

```diff
@@ -29,6 +23,7 @@ declare -r JMX_OPTS="-Dcom.sun.management.jmxremote \
   -Djava.rmi.server.hostname=$ADDRESS"
 declare -r HEAP_OPTS="${HEAP_OPTS:-"-Xmx2G -Xms2G"}"
 declare -r BROKER_PROPERTIES="/tmp/server-${BROKER_PORT}.properties"
+. ./init.sh
```

Comment:
Could you use `source` command?

Also, users could call the script at any working directory, so you have to locate the script path.